File: exercises/00 my first unit test/cs/My First Unit Test/StrangeNumber.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace My_First_Unit_Test
{
    public class StrangeNumber
    {
        private double value = 0.0;

        public double GetDouble()
        {
            return value;
        }

        public void SetDouble(double value)
        {
            this.value = value;
        }

        public void Add(double i)
        {
            value += i;
        }

        public void Subtract(double i)
        {
            value -= i;
        }

        public void Divide(double i)
        {
            value /= i;
        }

        public void Multiply(double i)
        {
            value *= i;
        }

    }
}

File: exercises/00 my first unit test/java/src/eu/sioux/tdd/myfirst/StrangeNumber.java
package eu.sioux.tdd.myfirst;

public class StrangeNumber {
    private double value = 0.0;

    public double getDouble() {
        return this.value;
    }

    public void setDouble(double value) {
        this.value = value;
    }

    public void add(double i) {
        this.value += i;
    }

    public void subtract(double i) {
        this.value -= i;
    }

    public void divide(double i) {
        this.value /= i;
    }

    public void multiple(double i) {
        this.value *= i;
    }
}
